src/media/css/reviews.styl

```diff
@@ -135,6 +137,21 @@ mkt-prompt .simple-field.rating {
     &.stars-0 { background-position: -65px 0; }
 }
 
+.stars .full {
+    color: #4cb1ff;
+    display: inline-block;
+
+    &.stars-4 { margin-left: -13px; }
+    &.stars-3 { margin-left: -26px; }
+    &.stars-2 { margin-left: -39px; }
+    &.stars-1 { margin-left: -52px; }
+    &.stars-0 { margin-left: -65px; }
+}
+.stars .empty {
+    color: #cbcbcb;
```

Comment:
We have these colors defined as variables here: https://github.com/mozilla/fireplace/blob/master/src/media/css/lib/colors.styl

Would you mind using the variables instead?